src/routes/rules/RulesView.vue

```diff
@@ -31,10 +31,12 @@
       <!-- Tutorial -->
       <v-row class="flex-column align-start mt-5">
         <h1 class="gradient-text">
-          {{ t('rules.tutorialTitle') }}
+          {{ t('rules.rulesTitle') }}
         </h1>
         <p>
-          {{ t('rules.tutorialText') }}
+          {{ t('rules.rulesReadText') }} 
+          <a href="https://github.com/cuttle-cards/cuttle/files/13283371/cuttle_rules.pdf"> Cuttle Cheestsheet </a>
```

Comment:
This is the right idea, but we should make a few changes to this link:
1. There was a problem with the pdf when the issue was initially written, we should replace this file with the new one that's been added to the issue just now. Apologies for the confusion there.
2. The pdf should be added to the source code as `/public/img/cuttle_rules.pdf` and then this link's `href` should be updated to be `href="/img/cuttle_rules.pdf"` so that the link points to the file hosted by the cuttle server, not github
4. We should add about="_blank" to make the link open in a new tab
5. The text "Cuttle Cheatsheet" should be translated into spanish and french and added to the "en.json", "es.json", and "fr.json" files so that it can be translated into the user's selected language.
Altogether then, it should look like this:
```suggestion
          <a href="/img/cuttle_rules.pdf" target="_blank"> {{ t('rules.cuttleCheatsheet') }} </a>
```